book_extractors/karelians/extraction/extractors/tests/test_war_data_extraction.py

```diff
@@ -158,3 +159,41 @@ def should_return_false_if_text_contains_mention_of_having_served_with_luksessa_
         self._verify_flags([
             ('Rautateiden Partaveitsi on ollut vuodesta -48 lähtien valtion rautateiden palveluksessa.', False)
         ], extractor)
+
+
+class TestLottaActivityFlagExtraction:
+    @pytest.yield_fixture(autouse=True)
+    def extractor(self):
+        return LottaActivityFlagExtractor(None, None)
+
+    def _verify_flags(self, expected_flags_and_texts, extractor):
```

Comment:
Hmm, kannattaako tätä kopioida erikseen joka suiteen? Voisiko tehdä hieman geneerisemmän version tästä jolla saisi nuo parametrit yms. konffattua siten, että tässä tiedostossa voisivat kaikki luokat käyttää testeissään samaa funktiota testaamiseen? 

Voisi joko toteuttaa erillisillä parametreilla, tai jos tuntuu hauskalta, niin jonain higher order funktiona, joka palauttaa valmiiksi konffatun funktion jonka scopessa on ne parametrit jotenkin tyyliin:
```
def get_verifier(flag_name, extractor):
    def verify(expected_flags_and_texts):
         # tänne nuo aiemmat kamat, sillä täältä scopesta näkee tuon extractorin
    return verify
```
Sitten generoi jokaiseen suiteen sille sopivan `verifier`:in ja kutsuu sitä sitten testeissä. En nyt ajatellut loppuun asti onko paras ratkaisu, mutta lähinnä ajatuksen ruoaksi jos tällainen patterni (higher order functions) ei ole tuttu konsepti.